test/integration/scope-hoisting/commonjs/require-re-export-default/b.js

```diff
@@ -1,2 +1,2 @@
-export foo from './c';
+export {default as foo} from './c';
```

Comment:
Not sure if `export foo from './c'` is valid, Babylon parses it but Babel throws. VSCode also shows invalid syntax errors.